sphinx-docs/usermanual/userman_admin.rst

```diff
@@ -6,24 +6,33 @@ Admininistrator User Manual
 A person who :
     * helps to plan and oversee a project using KA Lite
     * installs and updates KA Lite
-    * can create coach logins, student logins, download videos and language packs
+    * can create coach logins, Learner logins, download videos and language packs
 
 Administrator Glossary
 -------------------------------------------
 For users that choose to register online, there are some important terms to familiarize yourself with so that you understand how the flow of data works between installations and the online data hub.
 
 **Sharing Network**
-	A sharing network is a group of devices that share data between one another.
+	A sharing network is a group of devices that share data between one another. Data can mean video content, access software applications, and certain files.
```

Comment:
I don't think sharing networks will sync videos between one another. I'm also not too sure what you mean by software applications. Might be good to ask the dev room what kinda data is shared, but I think it's mostly stuff like learner/coach account information and progress.